.templates/auto-update.sh.template

```diff
@@ -0,0 +1,21 @@
+{% extends "templates/base_script.sh.template" %}
+
+{% block script %}
+    COMMIT_MSG="$(git log -1 --pretty=%B)"
+    git config --global user.email "info@appliedbrainresearch.com"
+    git config --global user.name "Nengo Bones"
+
+    {% for repo in repos %}
+    exe git clone "https://$GH_TOKEN@github.com/{{ repo }}" {{ repo }}
+    cd {{ repo }} || exit
+    exe bones-generate
+    if git diff HEAD --exit-code; then
+        echo "No changes"
+    else
+        exe git add --all
+        exe git commit -m "Automatic Nengo Bones update" -m "$COMMIT_MSG"
```

Comment:
I'm thinking we should change this to `git commit -m "Automatic update to Nengo Bones $COMMIT_HASH" -m "$COMMIT_MSG"`, where `COMMIT_HASH` is the abbreviated commit hash of the latest Nengo Bones commit. This would be a bit more in line with our current way of naming the updates (with "Update to Nengo Bones v0.0.0"), and makes it easier to tell when looking at an abbreviated log where it's up to date with.

Reply:
Knowing the commit hash wouldn't be very useful information to me. Like I just can't think of any scenario where I'd be looking at the commit hash in the `nengo/nengo` git history and trying to cross-reference that with the `nengo/nengo-bones` git history. So my slight preference is to avoid it just because it makes the commit history a bit messier, but if other people do have workflows where that'd be useful information to them then I'm fine with it.

Reply:
I have a `git lg` alias that does a one-line-per-commit summary of the log, so it'd be nice for me to be able to see the abbreviated hash there e.g. if I want to quickly double-check that a repo is up to date with the latest NengoBones. Also, it makes it possible to differentiate between the auto-update commits just from my `git lg` view. Because I often use that view, I really try to avoid commits with the same subject line.